src/Beberlei/Metrics/Collector/Null.php

```diff
@@ -13,6 +13,9 @@
 
 namespace Beberlei\Metrics\Collector;
 
+/**
+ * @deprecated for PHP7 compatibility, use NullCollector instead.
+ */
 class Null implements Collector
```

Comment:
Symfony already triggers a deprecation error so I though there was no need for an other one.
What would be the benefit of extending? To me, for a dummy implementation, there is no need.